### ISSUE TITLE
2.6.564: ImageLoader#getImage:21 - java.lang.IllegalArgumentException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#10106]
## Log Message
File must exist at path: /usr/share/triplea/bin/assets/launch_screens/triplea-logo.png

## TripleA Version
2.6.564

## Java Version
17.0.1

## Operating System
Linux

## Stack Trace
```
Exception: java.lang.IllegalArgumentExceptionFile must exist at path: /usr/share/triplea/bin/assets/launch_screens/triplea-logo.png
java.lang.Exception
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
	at org.triplea.io.ImageLoader.getImage(ImageLoader.java:21)
	at games.strategy.triplea.ResourceLoader.loadImageAsset(ResourceLoader.java:75)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorPanel.<init>(GameSelectorPanel.java:91)
	at games.strategy.engine.framework.startup.ui.panels.main.MainPanelBuilder.buildMainPanel(MainPanelBuilder.java:25)
	at games.strategy.engine.framework.ui.MainFrame.<init>(MainFrame.java:45)
	at games.strategy.engine.framework.ui.MainFrame.buildMainFrame(MainFrame.java:54)
	at games.strategy.engine.framework.GameRunner.lambda$start$0(GameRunner.java:66)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)


```

**Comment by DanVanAtta** [comment]
This was likely from a developer : )

The error here happens when the 'downloadAssets' gradle task is not run before launching the game. Generally that happens automatically by using the checked in launcher.

Typically the logo image is bundled with the game, and if game bundling is a problem, this is not necessarily the first error you would see in that case (hence it is likely a dev issue). 

The error message could be improved though to be directed to a developer to mention that they should run the checked in launcher, or at least run 'downloadAssets'. We should double check that the developer documentation leads a person to a path where this wouldn't be a problem.

**Comment by jtkenny** [comment]
Hey, I'm Justin the ISU student. Our group is gonna work on this issue for our first mini pull request.

**Comment by AndiLilaj** [comment]
>The error message could be improved
Could I just add + "You should build with the checked in launcher, or at least run 'downloadAssets'." to the original method errorMessage?

**Comment by AndiLilaj** [comment]
I guess it could be counted as closed with #11231
